provider/server/machine_volume_attach_test.go

```diff
@@ -7,26 +7,27 @@ import (
 	"time"
 
 	"github.com/digitalocean/go-libvirt"
-	iri "github.com/ironcore-dev/ironcore/iri/apis/machine/v1alpha1"
-	irimeta "github.com/ironcore-dev/ironcore/iri/apis/meta/v1alpha1"
+	machineiriv1alpha1 "github.com/ironcore-dev/ironcore/iri/apis/machine/v1alpha1"
+	irimetav1alpha1 "github.com/ironcore-dev/ironcore/iri/apis/meta/v1alpha1"
+	volumeiriv1alpha1 "github.com/ironcore-dev/ironcore/iri/apis/volume/v1alpha1"
 	libvirtutils "github.com/ironcore-dev/libvirt-provider/pkg/libvirt/utils"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"libvirt.org/go/libvirtxml"
 )
 
 var _ = Describe("AttachVolume", func() {
-	It("should correctly attach volume to machine", func(ctx SpecContext) {
+	It("should correctly attach an empty disk to the machine", func(ctx SpecContext) {
 		By("creating a machine")
-		createResp, err := machineClient.CreateMachine(ctx, &iri.CreateMachineRequest{
-			Machine: &iri.Machine{
-				Metadata: &irimeta.ObjectMetadata{
+		createResp, err := machineClient.CreateMachine(ctx, &machineiriv1alpha1.CreateMachineRequest{
+			Machine: &machineiriv1alpha1.Machine{
+				Metadata: &irimetav1alpha1.ObjectMetadata{
 					Labels: map[string]string{
 						"foo": "bar",
 					},
 				},
-				Spec: &iri.MachineSpec{
-					Power: iri.Power_POWER_ON,
+				Spec: &machineiriv1alpha1.MachineSpec{
```

Comment:
i recommend use squashfs and properly booted os. it is more close to our production environment. Here are you doing test with "unsupported" setup.